src/Management/src/EndpointBase/HeapDump/HeapDumpEndpointOptions.cs

```diff
@@ -24,5 +25,10 @@ public HeapDumpEndpointOptions(IConfiguration config)
                 Id = "heapdump";
             }
         }
+
+        public string HeapDumpType { get; set; }
+
+        // Default to disabled on Linux + Cloud Foundry until PTRACE is allowed
+        public override bool DefaultEnabled { get; } = !Platform.IsCloudFoundry || !Platform.IsLinux;
```

Comment:
This will change the default experience, but heap dump on CF + Linux can be re-enabled by using `management:endpoints:heapdump:enabled`

Reply:
What about when it is not CF and running on linux (outside of CF)?  It should be enabled then, right?

Reply:
Yes, it will be enabled any time it's not running on Cloud Foundry or if it's not running on Linux.

Reply:
Would `!(Platform.IsCloudFoundry && Platform.IsLinux);` be easier to understand?

Reply:
Ahh..  I read through it too quickly.  I like the updated code for readability.

Reply:
Yes - that is easier to reason about.